docs/guides/studio_oauth.rst

```diff
@@ -49,20 +45,11 @@ For each deployed environment (stage, production, etc.):
     SOCIAL_AUTH_EDX_OAUTH2_URL_ROOT: <server-to-server LMS root URL>  # possibly same as public LMS root URL
     SOCIAL_AUTH_EDX_OAUTH2_PUBLIC_URL_ROOT: <public LMS root URL>
 
-#. Configure Studio to initiative OAuth flow and use a separate session cookie::
+#. Configure Studio to initiate OAuth flow and use a separate session cookie::
 
-    LOGIN_URL: /login/  # to activate OAuth functionality
     SESSION_COOKIE_NAME: studio_sessionid
 
-Cleanup
--------
-
-Config and code changes to be performed after all environments are using OAuth flow for Studio.
-
-- Set ``LOGIN_URL`` to ``'/login/'`` in ``cms/envs/common.py``
-- Deploy
-- Remove ``LOGIN_URL`` overrides from all environments (devstack and others)
-- Remove doc (convert to migration instructions for next release)
+This cookie renaming step is also a good opportunity to change ``SESSION_COOKIE_DOMAIN`` if desired, since it allows for a clean transition without having to worry about `flaky behavior driven by order-unstable cookie-sending <https://fwielstra.github.io/2017/03/13/fun-with-cookies-and-subdomains/>`_. Narrowing the domain (or removing it, to keep subdomains from seeing the cookie) may improve the security of your deployment, but the best option depends on the exact layout of your domain names and is beyond the scope of this document.
 
 Declining the migration
```

Comment:
Do we really want to include untested decline option? Or if we do, maybe we should add some warnings? A pointer to the DEPR? A note that this won't work in the future. A strong recommendation that they actually follow the new path?

Reply:
Oh, yeah, we can remove this as well!